package.json

```diff
@@ -1,236 +1,236 @@
+                    "type": "array",
+                    "items": {
+                        "anyOf": [
+                            {
+                                "type": "string"
+                            },
+                            {
+                                "enum": [
+                                    "json"
+                                ]
+                            }
+                        ]
+                    },
+                    "default": [
+                        "json"
+                    ]
+                },
+                "prettier.graphqlEnable": {
+                    "description": "Advanced feature. Enable and use graphql parser for those language ids. Restart required",
+                    "type": "array",
+                    "items": {
+                        "anyOf": [
+                            {
+                                "type": "string"
+                            },
+                            {
+                                "enum": [
+                                    "graphql"
+                                ]
+                            }
+                        ]
+                    },
+                    "default": [
+                        "graphql"
+                    ]
+                }
+            }
+        },
+        "jsonValidation": [
+            {
+                "fileMatch": ".prettierrc",
+                "url": "http://json.schemastore.org/prettierrc"
+            }
+        ],
+        "languages": [
+            {
+                "id": "json",
+                "filenames": [
+                    ".prettierrc"
+                ]
+            }
         ]
-      }
-    ]
-  },
-  "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./node_modules/vscode/bin/test"
-  },
-  "devDependencies": {
-    "@types/mocha": "^2.2.32",
-    "@types/node": "^7.0.39",
-    "@types/semver": "^5.3.31",
-    "mocha": "^2.3.3",
-    "typescript": "^2.4.2",
-    "vscode": "^1.1.4"
-  },
-  "dependencies": {
-    "prettier": "1.6.1",
-    "prettier-eslint": "^7.1.0",
-    "read-pkg-up": "2.0.0",
-    "semver": "^5.4.1"
-  }
+    },
+    "scripts": {
+        "vscode:prepublish": "tsc -p ./",
+        "compile": "tsc -watch -p ./",
+        "postinstall": "node ./node_modules/vscode/bin/install",
+        "test": "node ./node_modules/vscode/bin/test"
+    },
+    "devDependencies": {
+        "@types/mocha": "^2.2.32",
+        "@types/node": "^7.0.39",
+        "@types/semver": "^5.3.31",
+        "mocha": "^2.3.3",
+        "typescript": "^2.4.2",
+        "vscode": "^1.1.4"
+    },
+    "dependencies": {
+        "prettier": "1.7.0",
+        "prettier-eslint": "^8.1.1",
+        "read-pkg-up": "2.0.0",
+        "semver": "^5.4.1"
+    }
 }
```

Comment:
What happened here??

Reply:
Ok, get it. Was not formatted with NPM.
.prettierignore this file soon :-)

Reply:
Yeah it got formatted, I'm sorry :D